.ci/bump-version.sh

```diff
@@ -21,7 +21,12 @@ docker run --rm -t \
     bundle update elastic-apm"
 
 # Validate whether the agent version matches
-git diff --name-only -S"elastic-apm (${NEW_AGENT_VERSION})" | grep Gemfile.lock
+git diff --name-only -S"elastic-apm (${NEW_AGENT_VERSION})" | grep Gemfile.lock && found=1 || found=0
```

Comment:
```suggestion
git diff --name-only -S"elastic-apm (${NEW_AGENT_VERSION})" | grep -c Gemfile.lock
```

Reply:
I like it, although It does require the `|| true` too, so I'd say I leave it as is.